distribution/docs/src/main/resources/_contents/_using/using-catalog-search-ui-contents.adoc

```diff
@@ -1,5 +1,5 @@
 
-Introduction: The ${catalog-ui} represents the most advanced search interface available with ${branding}.
+Introduction: ${catalog-ui} represents the most advanced search interface available with ${branding}.
```

Comment:
Maybe. Probably not until https://github.com/codice/ddf/pull/1714 gets in. 

At minimum, the value of that property should be set as part of that PR.